modules/services.sh

```diff
@@ -83,8 +86,37 @@ function services {
     done
   else
     if [ -z "$command" ]; then
-      echo "ERROR: no command given"
-      exit 1
+      service_list=($(services available))
+      for service in "${service_list[@]}"
+      do
+        if [ $service == $service_name ]; then
```

Comment:
You can remove this for loop by using `check_available_services $service_name` instead, which does the same thing as you are trying to do with the for loop.

If you do replace it with this function, it means that you have to replace all the `${service}`s inside the body of the for loop with `$service_name` instead.